src/github.com/mozilla-services/pushgo/simplepush/handlers_endpoint.go

```diff
@@ -370,7 +374,7 @@ func (h *EndpointHandler) deliver(cn http.CloseNotifier, uaid, chid string,
 	// Increment the appropriate final metric whether deliver did or
 	// did not work
 	if delivered {
-		h.metrics.Increment("updates.appserver.received")
+		h.metrics.IncrementByRate("updates.appserver.received", 1, 0.1)
 	} else {
 		h.metrics.Increment("updates.appserver.rejected")
```

Comment:
Do we want to sample delivery failures, too? I imagine this would be more common in production, given the low connection lifetimes we've seen for Loop.